packages/jsii-pacmak/test/generated-code/__snapshots__/target-go.test.ts.snap

```diff
@@ -7443,7 +7320,7 @@ func (s *Statics) GetZooBar() map[string]string {
 }
 
 func (s *Statics) GetInstance() Statics {
-    return s.Instance
+    return *s.Instance
```

Comment:
@SoManyHs here is an example of the change to getters. This may only be necessary as long as the returns of getters are direct property accesses and not calls to the JSII runtime. But for now this is necessary because the `Instance` property of statics is a `*static` so we have to deref it to make it the proper return type.